resources/prometheus/prometheus-rules.yaml

```diff
@@ -175,8 +175,69 @@ spec:
+          annotations:
+            summary: "The number of RDS DB clusters is close to its limit."
+            description: "The number of AWS RDS DB clusters is close to its limit. A quota increase should be requested from AWS."
+            sop_url: "https://gitlab.cee.redhat.com/stackrox/acs-managed-service-runbooks/blob/master/sops/dp-029-increase-aws-limits.md"
+        - alert: RHACSCentralDBInstancesUtilizationHigh
+          expr: |
+            expr: acs_fleetshard_central_db_instances_used / acs_fleetshard_central_db_instances_max >= 0.8
+          for: 1h
+          labels:
+            severity: warning
+          annotations:
+            summary: "The number of RDS DB instances is close to its limit."
+            description: "The number of AWS RDS DB instances is close to its limit. A quota increase should be requested from AWS."
+            sop_url: "https://gitlab.cee.redhat.com/stackrox/acs-managed-service-runbooks/blob/master/sops/dp-029-increase-aws-limits.md"
+        - alert: RHACSCentralDBManualSnapshotsUtilizationHigh
+          expr: |
+            expr: acs_fleetshard_central_db_snapshots_used / acs_fleetshard_central_db_snapshots_max >= 0.8
+          for: 1h
+          labels:
+            severity: warning
+          annotations:
+            summary: "The number of RDS DB snapshots is close to its limit."
+            description: "The number of AWS RDS DB manual snapshots is close to its limit. A quota increase should be requested from AWS."
+            sop_url: "https://gitlab.cee.redhat.com/stackrox/acs-managed-service-runbooks/blob/master/sops/dp-029-increase-aws-limits.md"
+        - alert: RHACSCentralDBClustersUtilizationCritical
+          expr: |
+            expr: acs_fleetshard_central_db_clusters_used / acs_fleetshard_central_db_clusters_max >= 0.9
+          for: 1h
+          labels:
+            severity: critical
+          annotations:
+            summary: "The number of RDS DB clusters is very close to its limit."
+            description: "The number of AWS RDS DB clusters is close to its limit. A quota increase must be requested from AWS."
+            sop_url: "https://gitlab.cee.redhat.com/stackrox/acs-managed-service-runbooks/blob/master/sops/dp-029-increase-aws-limits.md"
+        - alert: RHACSCentralDBInstancesUtilizationCritical
+          expr: |
+            expr: acs_fleetshard_central_db_instances_used / acs_fleetshard_central_db_instances_max >= 0.9
+          for: 1h
+          labels:
+            severity: critical
+          annotations:
+            summary: "The number of RDS DB instances is very close to its limit."
+            description: "The number of AWS RDS DB instances is close to its limit. A quota increase must be requested from AWS."
+            sop_url: "https://gitlab.cee.redhat.com/stackrox/acs-managed-service-runbooks/blob/master/sops/dp-029-increase-aws-limits.md"
+        - alert: RHACSCentralDBManualSnapshotsUtilizationCritical
```

Comment:
Is the inability to delete an instance impact SLOs?

If having an instance stuck in the "deprovisioning" status is OK, this `critical` alert can be removed, and we can only have the `warning` alert.

Reply:
OK, I'll remove the critical warning.

We request a final snapshot of the DB when deleting it. This is absolutely needed in case of accidental deletion, because the automatic snapshots are all deleted along with the DB. And so, if there's no more quota for manual snapshots, the DB deletion will fail and the ACS instance will be stuck in the deprovisioning state.